release-calendar/src/client/stylesheets/calendarStyle.scss

```diff
@@ -0,0 +1,19 @@
+@import '~@fullcalendar/core/main.css';
+@import '~@fullcalendar/daygrid/main.css';
+@import '~@fullcalendar/timegrid/main.css';
+.style {
+  font-family: Arial, Helvetica Neue, Helvetica, sans-serif;
+}
+.demo-app {
+  font-family: Arial, Helvetica Neue, Helvetica, sans-serif;
+  font-size: 14px;
+}
+
+.demo-app-top {
+  margin: 0 0 3em;
+}
+
+.demo-app-calendar {
+  margin: 0 auto;
+  max-width: 900px;
```

Comment:
Looks small on my monitor (2518px x 1338px). Thoughts on turning this into a percentage?

Some good tips here - https://developers.google.com/web/fundamentals/design-and-ux/responsive

Reply:
Oh, cool, thank you!